Functions.Templates/Templates/HttpTrigger-CSharp-Isolated/.template.config/template.json

```diff
@@ -64,6 +64,17 @@
                 "projectFileExtensions": ".csproj"
             }
         },
+        {
+            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore Nuget package",
+            "ActionId": "n/a",
```

Comment:
Not sure where this value comes from ...

Reply:
Action Id is predefined guid from templating engine. You can copy this from another template.